warehouse-management-service/internal/config/config.go

```diff
@@ -38,3 +46,27 @@ func FromFile(path string) (*Config, error) {
 
 	return &config, nil
 }
+
+func FromEnv() (*Config, error) {
+	config := make(map[string]string)
+	for identifier, envVar := range environmentVariables {
+		value, ok := os.LookupEnv(envVar)
+		if !ok {
+			return nil, fmt.Errorf("unable to read environment variable: %s", envVar)
+		}
+		config[identifier] = value
+	}
+	return &Config{
+			Postgres: PostgresConfig{
+				Host:     config["dbHost"],
+				Port:     config["dbPort"],
+				Username: config["dbUsername"],
+				Password: config["dbPassword"],
+				DBName:   config["dbName"],
+				SSLMode:  config["dbSSLMode"],
```

Comment:
> Can we directly use the envvar name as the key

This is the approach I initially wanted to go ahead with, however, I chose not to because of the awkward workaround of creating a map of `map[string]struct{}`.
Adding docstrings is a good point I hadn't considered earlier. 

Made this change.